### DISCUSSION TITLE
dirac.cfg :: set the value of a non-default name FileCatalog

### DISCUSSION THREAD

**Discussion by adriansev** [DIRACGrid/DIRAC#6836] [Q&A] [answered]
Hi! Is there an option to add in dirac.cfg to set the name of file catalog?
In AUGER we use a non-default named FileCatalog and it is required to be specified in command line (or in aliases), so it would be fantastic if it can be specified in configuration...
Thanks a lot!

**Comment by andresailer** [comment, marked as answer]
Yes you can, as described here:
https://dirac.readthedocs.io/en/latest/AdministratorGuide/HowTo/dedicateddfc.html#configuration-settings

This should probably go not only in your client dirac.cfg but in the central configuration!

**Reply by adriansev** [reply]
Thanks a lot! Sorry that i wasn't clear, i meant for client only installation..
AFAIU we use a shared instance of central services, and we have our own distinct catalog.
And related to your link, my question was if it's possible to set in client dirac.cfg the catalog to be used to avoid usage of `-f` option in `dirac-dms-filecatalog-cli -f MySpecialCatalog`
Thanks a lot!

**Reply by andresailer** [reply]
You can try setting the operations thing in your dirac.cfg, but if no-one fixed dirac-dms-filecatalog-cli, then that doesn't help, but you can probably fix the script?

Even in the shared installation, you want to set that config, it will only affect the Auger VO, and you want all your jobs to register files in that filecatalog, so that config should go to the central configuration, I guess...

**Reply by adriansev** [reply]
i see that in services configuration the actually the correct setup:
```
Services
      {
        Catalogs
        {
          AugerLcgFileCatalog
          {
            AccessType = Read
            Status = Active
          }
          AugerFileCatalog
          {
            AccessType = Read-Write
            Status = Active
          }
          CatalogList = AugerFileCatalog
        }
      }
```
so, the instructions from https://dirac.readthedocs.io/en/latest/AdministratorGuide/HowTo/dedicateddfc.html#configuration-settings seems to be in place since a long time
and given that the link mention: 
`The command dirac-dms-add-file does figure out from the VO which filecatalog should be used`
and with your comment i get that `dirac-dms-filecatalog-cli` simply does not have the mechanics that dirac-dms-add-file have for auto-detecting the correct catalog and it should be done by someone interested :) .. got it, thanks! :)

**Reply by andresailer** [reply]
I hope you are the interested one!

**Comment by atsareg** [comment]
I think that #6855 PR should resolve this issue.
